.github/workflows/build.yaml

```diff
@@ -11,38 +11,125 @@ on:
     outputs:
       artifactUrl:
         description: "The url of the compressed artifacts"
-        value: ${{ jobs.compress-assets.outputs.artifactUrl }}
+        value: ${{ jobs.combine-assets.outputs.artifactUrl }}
 
 permissions:
   contents: read
 
-# env:
-#   TEST_IMAGE_NODE_MAJOR_VERSION: 22
-
 jobs:
-  compress-assets:
+  build-linux:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    strategy:
+      matrix:
+        arch: [arm64v8, amd64, i386, arm32v7]
+    steps:
+      - name: Set up QEMU dependency
+        uses: docker/setup-qemu-action@v3.6.0
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3.10.0
```

Comment:
Do you mind if we pin these to SHAs instead of versions? I'm hoping we can move to immutable actions once GH has it ready, but pinning to SHAs gives us slightly better security in the meantime 🙇‍♀️